third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -298,6 +386,15 @@ def to_string(
     ):
         """Render a DataFrame to a console-friendly tabular output.
 
+        **Examples:**
+
+            >>> import bigframes.pandas as bpd
+            >>> bpd.options.display.progress_bar = None
+
+            >>> df = bpd.DataFrame({'col1': [1, 2], 'col2': [3, 4]})
+            >>> df.to_string()
+            '   col1  col2\\n0     1     3\\n1     2     4'
```

Comment:
Would it not work if we kept the expected output multiline, encapsulated in """?